src/schema/messages.json

```diff
@@ -41,7 +41,7 @@
     "WebExtensionMessages": {
       "type": "object",
       "patternProperties": {
-        "^[A-Za-z0-9_@]+$": {
+        "^.+$": {
```

Comment:
:+1: this doesn't seem to have any particular check in Firefox and it looks like it is the exactly the property that #1918 would like to be free from restrictions applied by linter that are not actually applied on Firefox.